examples/additional_examples/active_uphill.py

```diff
@@ -0,0 +1,29 @@
+import gaptrain as gt
+import autode as ade
+from autode.wrappers.keywords import GradientKeywords, OptKeywords
+gt.GTConfig.n_cores = 8
+
+
+# TODO: remove
```

Comment:
Is this a 'wait until Pete fixes it problem?'

Reply:
good point - should be removed in the public version